implementing-nosto/implement-search/implement-search-using-api/implementing-autocomplete.md

```diff
@@ -22,7 +22,12 @@ query {
     accountId: "YOUR_ACCOUNT_ID"
     query: "green"
     products: { size: 5 },
-    keywords: { size: 5 }
+    keywords: { size: 5 },
+    categories: { size: 5 },
+    popularSearches: { 
+      size: 5,
+      emptyQueryMatchesAll: true
```

Comment:
❓ Is an API key required to use this parameter within `popularSearches`? Asking because an API key is required to do so within `products`.